src/main/java/world/bentobox/bentobox/database/objects/Island.java

```diff
@@ -940,4 +940,33 @@ public String getGameMode() {
     public void setGameMode(String gameMode) {
         this.gameMode = gameMode;
     }
-}
\ No newline at end of file
+    
+    /**
+     * Returns if this island has its end island generated.
+     * @return {@code true} if this island has its end island generated, {@code false} otherwise.
+     * @since 1.5.0
+     */    
+    public boolean hasEndIsland(){
+        if (!BentoBox.getInstance().getIWM().isEndGenerate(getWorld()) || !BentoBox.getInstance().getIWM().isEndIslands(getWorld()) || BentoBox.getInstance().getIWM().getEndWorld(getWorld()) == null ){
```

Comment:
BetnoBox.getInstance().getIWM().isIslandNether(getWorld()).
This would be better in this case

Reply:
Oh... you are overwriting location world.
So input world is not end. 
Nop, you cannot use method I provide.